src/librustc_privacy/lib.rs

```diff
@@ -1131,14 +1118,10 @@ impl<'a, 'tcx> SanePrivacyVisitor<'a, 'tcx> {
                     check_inherited(tcx, i.span, i.vis);
                 }
             }
-            hir::ItemEnum(ref def, _) => {
```

Comment:
Ah by the way, I want a super review on this removal by somebody familiar with `rustc_privacy`. I’m not exactly super familiar with `rustc_privacy` and since no tests got broken, I assume it is fine.